modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -27,11 +27,11 @@ In the table, features are marked with the following statuses:
 
 s|Version s|kam  s|Argo CD CLI  s|Helm  s|Kustomize s|Argo CD s|Argo Rollouts s|Dex     s|RH SSO |
 
+|1.15.0 |NA  |2.13.1 TP |3.15.4 GA |5.4.3 GA |2.13.1 GA |1.7.2 TP |2.41.1 GA |7.6.0 GA |4.14-4.17
+
```

Comment:
For 1.14 and 1.13, lets have the shipped component version instead of `NA` in the compatibility matrix. Based on [older release note](https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/1.14/html/release_notes/gitops-release-notes#GitOps-compatibility-support-matrix_gitops-release-notes), it will be `0.0.51 TP`.

Reply:
Fixed.